recipes-appfw/sample-node0/files/manifest.in

```diff
@@ -0,0 +1,21 @@
+{
+    "application": "node0",
+    "description": "node0 - A NodeJS sample app.",
+    "provider": "test",
+    "groups": "test-node0",
+    "environment": {
+        "NODE0_FOO": "node0-foo",
+        "NODE0_BAR": "node0-bar",
+    },
+    "command": [
```

Comment:
hum... while this looks reasonable, I'm thinking if we're looking at the wrong place.

When systemd starts, it will look into the `default.target` and pull what it depends. Then we do not need to disable anything if we do not put it in our container-`default.target`. That means we should create a `default.target` in our container `/run` and let it pull what is wanted.

What do you think?